analytics_data_api/v0/views/learners.py

```diff
@@ -251,33 +267,33 @@ class CourseLearnerMetadata(CourseViewMixin, generics.RetrieveAPIView):
 
     **Response Values**
 
-        Returns a JSON object with the following keys:
+        Returns a JSON object with the following keys.
 
-            * cohorts: An object mapping the names of cohorts in the course to
-              the number of students belonging to those cohorts.
-            * segments: An object mapping the names of segments in the course
-              to the number of students belonging to those segments.  The
-              current set of segments are: "highly_engaged", "disengaging",
+            * cohorts: An object that maps the names of cohorts in the course
+              to the number of learners belonging to those cohorts.
+            * segments: An object that maps the names of segments in the course
+              to the number of learners belonging to those segments. The
+              current set of segments is "highly_engaged", "disengaging",
               "struggling", "inactive", and "unenrolled".
```

Comment:
For values in data, I put quotes around the data value only, rather than punctuating as though this was a spoken sentence that I was quoting. Which I admit is a difference of opinion with the MS manual of style, which states "placement of the closing quotation mark depends on whether the punctuation is part of the material being quoted." but then shows only examples with punctuation inside quotation marks as being MS style...